packages/mds-db/index.ts

```diff
@@ -297,7 +297,7 @@ async function updateDevice(device_id: UUID, changes: Partial<Device>): Promise<
 }
 
 async function writeEvent(event_param: VehicleEvent) {
-  await readDevice(event_param.device_id)
+  await readDevice(event_param.device_id, true)
```

Comment:
So, is this just to make sure the device exists such that it will throw an exception if if doesn't? We could use FKs to enforce this sort of referential integrity.

Reply:
We should probably be using the cache more but it's probably best to wait till after we move to Kubernetes.